### PR TITLE
Update ocluster, ocurrent and obuilder-spec.

### PR DESCRIPTION
**Pull request by tmcgilchrist**

### CODE REVIEW

File: ocaml-docs-ci.opam.template
Comment (tmcgilchrist) [inline ocaml-docs-ci.opam.template:2]: This can point to https://github.com/ocaml/opam-repository/pull/22467 once it's released.

**Review by maiste — COMMENTED** [review]
LGTM :+1: 
Shouldn't we wait for the PR to be merged in `opam` before merging?

**Review by TheLortex — APPROVED** [review]

**Comment by maiste** [comment]
I think we can merge it even if the macOS worker is not ok as we are going to run it on a Linux server :)

**Pull request merged**
